src/commctl/cli.py

```diff
@@ -150,6 +150,42 @@ def __init__(self, conf):
         self._con.headers['Content-Type'] = 'application/json'
         self._con.auth = (conf['username'], conf['password'])
 
+    def _handle_status(self, resp, operation):
+        """
+        Generic handling of status responses.
+
+        :param resp: The response to look at.
+        :type resp: requests.Response
+        :param operation: The name of the operation. EX: get
+        :type operation: str
```

Comment:
You could eliminate the `operation` argument and use `resp.request.method`.  Slightly cleaner.